packages/eslint-plugin-pf-codemods/src/rules/v6/labelRemoveIsOverflowLabel/label-remove-isOverflowLabel.md

```diff
@@ -0,0 +1,17 @@
+### label-remove-isOverflowLabel [(#10037)](https://github.com/patternfly/patternfly-react/pull/10037)
+
+isOverflowLabel prop for Label has been replaced with variant="overflow"
```

Comment:
Related to above comment, we should add onto the message here + when an error is thrown: "Running the fix for this rule will replace an existing `variant` prop." or something along those lines.